src/github.com/mihard/behat-parallel-runner/runner/scenario.go

```diff
@@ -19,17 +21,21 @@ func Worker(wn int, sc chan Scenario, rc chan Result) {
 }
 
 func run(wn int, s Scenario) Result {
+	start := time.Now()
 	cmd := exec.Command(BEHAT, s.File, fmt.Sprintf("--name=%s", s.Scenario))
+	executionTime := time.Since(start)
```

Comment:
On this line, the command is not yet executed. Please move the expression under `cmd.Output()` (line 28)